Makefile

```diff
@@ -62,7 +62,7 @@ install:
 
 .PHONY: test
 test:
-	pytest -n auto -v --runslow --cov=pyquil test/unit
+	pytest -n 1 -v --runslow --cov=pyquil test/unit
```

Comment:
:{

Reply:
@notmgsk This won't stay. Part of a debugging commit only ;)